src/Hl7.Fhir.Specification/data/account.xsd

```diff
@@ -27,7 +27,7 @@
   POSSIBILITY OF SUCH DAMAGE.
   
 
-  Generated on Tue, Mar 21, 2017 06:09+1100 for FHIR v3.0.0 
+  Generated on Wed, Apr 19, 2017 07:44+1000 for FHIR v3.0.1 
```

Comment:
Ah we shipped the pre-technical correction XSDs....no one ever complained ;-)

Reply:
Yes, indeed. I also noticed this, when I was doing the downloads again